.github/workflows/arcade-sync.yml

```diff
@@ -75,5 +80,8 @@ jobs:
         commit-message: "Sync shared code from ${{ env.SourceRepoName }}"
         labels: area-infrastructure
         path: .\${{ env.TargetRepoName }}
+        # May have problems here with a team name. team-reviewers property doesn't work due to
+        # https://github.com/peter-evans/create-pull-request/issues/155 Avoiding that for now.
+        reviewers: dougbu, wtgodbe
```

Comment:
yes, I'd rather use `dotnet/aspnet-build` here but adding a secret to the GitHub token seems worse